### ISSUE TITLE
Focus loss in content navigator.

### ISSUE THREAD

**Issue by dh256git** [dh256git/project27#3]
## Describe the bug
The screen reader focus jumps away when opening the content navigator button in the secondary navigation.

## Help us reproduce the bug

Steps to reproduce the behaviour:

1. Go to 'Any page'
2. Click on 'the bookmark icon (contents)'

## Expected behaviour

The focus of the screen reader cursor should be at the beginning of the drop down menu of contents.

## System

What system have you used?

**Desktop (please complete the following information):**

 - OS: [MacOS]
 - Browser [safari]
 - Version [16]
